tensorbay/client/dataset.py

```diff
@@ -190,6 +190,14 @@ def _delete_branch(self, name: str) -> None:
 
         self._client.open_api_do("DELETE", "branches", self.dataset_id, json=delete_data)
 
+    def import_alldata_by_prefix(self, path: str, segment_name: str) -> None:
+        """Import all files in cloud platform path to a specific segment.
```

Comment:
Import all files prefixed by the given cloud platform path to a specific segment.